float32[5] angle_fb
int8 bufferHealth